app/tasks/kis.py

```
@@ -0,0 +1,715 @@
+                    stock_steps.append({'step': '매수', 'result': res})
+                except Exception as e:
+                    stock_steps.append({'step': '매수', 'result': {'success': False, 'error': str(e)}})
+
+                # 3. 매도
+                self.update_state(state='PROGRESS', meta={'status': f'{name} 매도 주문 중...', 'current': index, 'total': total_count, 'percentage': int((index / total_count) * 100)})
+                try:
+                    # 매수 후 잔고가 변했을 수 있으므로 다시 조회하거나, 기존 수량 사용 (여기선 기존 수량 사용)
+                    # 실제로는 매수 체결 확인이 필요하지만, 비동기라 즉시 반영 안될 수 있음.
+                    res = await process_kis_domestic_sell_orders_with_analysis(kis, code, current_price, avg_price, qty)
+                    stock_steps.append({'step': '매도', 'result': res})
+                except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)